robottelo/constants.py

```diff
@@ -660,46 +667,46 @@
 
 CUSTOM_RPM_SHA_512_FEED_COUNT = {'rpm': 35, 'errata': 4}
 
-CUSTOM_MODULE_STREAM_REPO_1 = 'https://partha.fedorapeople.org/test-repos/pteradactyl/'
-CUSTOM_MODULE_STREAM_REPO_2 = 'https://partha.fedorapeople.org/test-repos/rpm-with-modules/el8/'
-CUSTOM_SWID_TAG_REPO = 'https://partha.fedorapeople.org/test-repos/swid-zoo/'
+CUSTOM_MODULE_STREAM_REPO_1 = f'{REPOS_URL}/module_stream1'
+CUSTOM_MODULE_STREAM_REPO_2 = f'{REPOS_URL}/module_stream2'
+CUSTOM_SWID_TAG_REPO = f'{REPOS_URL}/swid_zoo'
 CUSTOM_REPODATA_PATH = '/var/lib/pulp/published/yum/https/repos'
 CERT_PATH = "/etc/pki/ca-trust/source/anchors/"
-FAKE_0_YUM_REPO = 'http://inecas.fedorapeople.org/fakerepos/zoo/'
-FAKE_1_YUM_REPO = 'http://inecas.fedorapeople.org/fakerepos/zoo3/'
-FAKE_2_YUM_REPO = 'http://inecas.fedorapeople.org/fakerepos/zoo2/'
-FAKE_3_YUM_REPO = 'http://omaciel.fedorapeople.org/fakerepo01'
-FAKE_4_YUM_REPO = 'http://omaciel.fedorapeople.org/fakerepo02'
+FAKE_0_YUM_REPO = f'{REPOS_URL}/fake_yum0'
+FAKE_1_YUM_REPO = f'{REPOS_URL}/fake_yum1'
+FAKE_2_YUM_REPO = f'{REPOS_URL}/fake_yum2'
+FAKE_3_YUM_REPO = f'{REPOS_URL}/fake_yum3'
+FAKE_4_YUM_REPO = f'{REPOS_URL}/fake_yum4'
 FAKE_5_YUM_REPO = 'http://{0}:{1}@rplevka.fedorapeople.org/fakerepo01/'
-FAKE_6_YUM_REPO = 'https://stephenw.fedorapeople.org/fakerepos/needed-errata/'
-FAKE_7_YUM_REPO = 'https://fixtures.pulpproject.org/rpm-long-updateinfo/'
-FAKE_8_YUM_REPO = 'https://abalakht.fedorapeople.org/test_repos/lots_files/'
-FAKE_9_YUM_REPO = 'https://stephenw.fedorapeople.org/fakerepos/multiple_errata/'
-FAKE_10_YUM_REPO = 'https://partha.fedorapeople.org/test-repos/separated/modules-rpms/'
-FAKE_11_YUM_REPO = 'https://partha.fedorapeople.org/test-repos/separated/rpm-deps/'
-FAKE_YUM_DRPM_REPO = 'https://fixtures.pulpproject.org/drpm-signed/'
-FAKE_YUM_SRPM_REPO = 'https://fixtures.pulpproject.org/srpm-signed/'
-FAKE_YUM_SRPM_DUPLICATE_REPO = 'https://fixtures.pulpproject.org/srpm-duplicate/'
-FAKE_YUM_MIXED_REPO = 'https://pondrejk.fedorapeople.org/test_repos/mixed/'
+FAKE_6_YUM_REPO = f'{REPOS_URL}/needed_errata'
+FAKE_7_YUM_REPO = 'https://repos.fedorapeople.org/pulp/pulp/demo_repos/large_errata/zoo/'
+FAKE_8_YUM_REPO = f'{REPOS_URL}/lots_files'
+FAKE_9_YUM_REPO = f'{REPOS_URL}/multiple_errata'
+FAKE_10_YUM_REPO = f'{REPOS_URL}/modules_rpms'
+FAKE_11_YUM_REPO = f'{REPOS_URL}/rpm_deps'
+FAKE_YUM_DRPM_REPO = 'https://repos.fedorapeople.org/repos/pulp/pulp/fixtures/drpm/'
+FAKE_YUM_SRPM_REPO = 'https://repos.fedorapeople.org/repos/pulp/pulp/fixtures/srpm/'
+FAKE_YUM_SRPM_DUPLICATE_REPO = 'https://repos.fedorapeople.org/pulp/pulp/fixtures/srpm-duplicate/'
```

Comment:
Thanks for the Comment @JacobCallahan .

They are replaced as below. Late me know, if thats incorrect!

```suggestion
FAKE_YUM_DRPM_REPO = 'https://fixtures.pulpproject.org/drpm-unsigned/'
FAKE_YUM_SRPM_REPO = 'https://fixtures.pulpproject.org/srpm-unsigned/'
FAKE_YUM_SRPM_DUPLICATE_REPO = 'https://fixtures.pulpproject.org/srpm-duplicate/'
```